src/renderer/pages/Staking/Operations/components/ValidatorsTable/ValidatorsTable.tsx

```diff
@@ -0,0 +1,91 @@
+import { Address, Asset, Explorer, Validator } from '@renderer/shared/core';
+import { cnTw, getComposedIdentity } from '@renderer/shared/lib/utils';
+import { BodyText, FootnoteText, HelpText, Icon, Identicon, InfoPopover } from '@renderer/shared/ui';
+import { AssetBalance } from '@renderer/entities/asset';
+import { AssetFiatBalance } from '@renderer/entities/price/ui/AssetFiatBalance';
+import { useI18n } from '@renderer/app/providers';
+import { ExplorerLink } from '@renderer/components/common';
+
+const TABLE_GRID_CELLS = 'grid-cols-[1fr,128px,128px,40px]';
+
+type Props = {
```

Comment:
To be more specific as we do few lines below in `RowProps`
```suggestion
type TableProps = {
```